### DISCUSSION TITLE
Ableton Live 10 -> Zoom not working on M1 Mini

### DISCUSSION THREAD

**Discussion by jjgranger** [ExistentialAudio/BlackHole#272] [Q&A]
Hi, I'm running build 0.2.8 on a new M1 Mac mini running Big Sur. Trying to route Ableton Live 10 to Zoom (latest version). 

I set up the multi-output device in MIDI setup, routed Live's output to it and set Zoom's mic to BlackHole 16ch. No signal is getting to Zoom. Using the same setup on my 2014 MacBook (running Mojave) had no problems. 

When I open the 16ch in the Sound pref Input on the Mini, I can see audio signal being routed to it from Live. Just doesn't seem to get to Zoom? I have made sure that Zoom has permissions to use the mic, and I have made sure that the External Headphones device is set as the top level master in the Multi-Ouput Device. Uninstalled and reinstalled Black Hole. Restarted all programs, and the mini. Still nothing to Zoom.

Any ideas would be much appreciated...thanks!

**Comment by devinroth** [comment]
Sounds like you are doing everything right. But please upload some screenshots. Maybe you’re missing something simple.

Devin

**Reply by jjgranger** [reply]
Managed to figure it out after a ton of random clicking around. Looks like Zoom's latest update is possibly more aggressive with the background noise cancellation — or perhaps what I was sending it sounds more like noise — so the "Suppress background noise" setting in Zoom audio prefs needed to be changed from "Auto" to "Low". That did it!

![Screen Shot 2021-01-25 at 11 03 59 AM](https://user-images.githubusercontent.com/77858918/105731672-a3a26d80-5efd-11eb-9901-714d9a40ff30.png)

**Comment by devinroth** [comment]
Ah good to know. Glad you figured it out.

Devin
